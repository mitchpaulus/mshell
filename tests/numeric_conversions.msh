"9.999" toFloat? 2 toFixed wl
"10e8" toFloat? str wl
"1e-8" toFloat? str wl
"1.234e4" toFloat? str wl
"1.234E4" toFloat? str wl

"basd" toFloat str wl # Should be None

"0001" toInt? 1 = str wl
"1.1" toInt str wl # Should be None
