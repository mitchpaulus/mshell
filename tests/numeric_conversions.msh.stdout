10.00
1000000000
0.00000001
12340
12340
None
true
None
